### PR TITLE
Update docs.en.md

### PR DESCRIPTION
**Pull request by TheHasagi**
Regarding #369

### CODE REVIEW

**Comment by TheHasagi** [comment]
Guys, please check for correctness. @admitrevskiy  you should approve that technical explanation of option is correct. @vbagirov I need your help for a better translation and typo fixes.

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 1]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+   
+If true, VPN will be configured to bypass all the IPv4 traffic.
+
+   **pref.vpn.ipv6.bypass**
+
+If true, VPN will be configured to bypass all the IPv6 traffic.
+
+   **pref.vpn.ipv6.disable**
+
+If true - AdGuard will disable IPv6 routes forcibly.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: This is true for VPN mode only

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 2]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+   
+List of IPv6 routes excluded from VPN and auto-proxy.
+
+   **pref.excluded.uids**
+
+List of excluded packages uids.
+
+  **pref.vpn.capture**
+
+Enable VPN tun device logging.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: I think we need more explanation here.
Please, add some info about `PCAP` file

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 3]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+Enable HAR file capture.
+
+  **pref.vpn.disable.pause**
+   
+Disable automatic VPN pause in case of network absence, tethering or power saving mode.
+
+  **pref.vpn.disable.reconfigure**
+
+Disables VPN automatic re-configuration.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: When does it happen? It's not clear, is it?

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 4]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+  **pref.proxy.disable.reconfigure**
+
+Disables VPN automatic re-configuration.
+
+  **pref.proxy.block.ipv6**
+
+Block all internet connections through IPv6.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: For root proxy only

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 5]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+List of the redirect ports.
+
+  **pref.boot.startup.delay**
+
+The initial protection startup delay (in seconds) after device boot up.
+
+  **pref.enforce.paused.notification**
+
+Enforce notification about paused protection even when notification icon is set to None.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: What is `None`? This part is actual for Android versions below Oreo.

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 6]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+  **pref.root.clear.youtube**
+  
+AdGuard will clear Youtube app data every on boot.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: Root needed

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 7]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+  **pref.root.set.oom_adj**
+   
+AdGuard will set minimum oom_score_adj for own process.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: Root needed

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 8]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+   
+List of apps for which AdGuard enforces HTTPS filtering even if they target Android 7+.
+
+  **pref.removed.html.log**
+
+AdGuard will print information about removed HTML elements to the filtering log.
+    
+  **pref.samsungpay.autopause.enable**
+
+AdGuard will pause protection while Samsung Pay app is in the foreground.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: In VPN mode only

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 9]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+  **pref.dns.bootstrap**
+   
+ Bootstrap DNS for DNSProxy DoH and DoT servers.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: What is `DNSProxy`?

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 10]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
 This setting has been developed specifically for the South Korean users who often face the following problem — ***the "Samsung Pay" app doesn’t work on devices with running VPN** (which, in its turn, is being used by AdGuard).* 
 
 If you want to learn about how this feature makes AdGuard compatible with Samsung Pay, please read this [article](https://kb.adguard.com/en/android/solving-problems/samsung-pay).
+
+
+### All available options:
+
+  **pref.vpn.ipv4.force.default**
+   
+If enabled - AdGuard will force IPv4 default route.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: `VPN will be configured to filter 0.0.0.0/0.`

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 11]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+IPv4 TUN interface address.
+
+   **pref.vpn.ipv6.address**
+
+IPv6 TUN interface address.
+
+   **pref.vpn.ipv6.force**
+   
+If enabled - AdGuard will force IPv6 default route (if IPv6 is available).
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: `add an IPv6 address to the VPN even if there's no active IPv6 network`

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 12]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+Список маршрутов IPv4, исключенных из VPN и авто-прокси.
+   
+   **pref.ipv6.routes.excluded**
+   
+Список маршрутов IPv6, исключенных из VPN и авто-прокси.
+
+   **pref.excluded.uids**
+
+Исключенные UID'ы.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: пакетов

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 13]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+Начальная задержка запуска защиты (в секундах) после загрузки устройства.
+
+  **pref.enforce.paused.notification**
+
+Принудительное уведомление о приостановленной защите, даже если для значка уведомления установлено значение Выключено.
+
+  **pref.root.clear.youtube**
+  
+AdGuard будет очищать данные приложения Youtube каждый раз при загрузке.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: Рут

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 14]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+  **pref.root.set.oom_adj**
+   
+AdGuard установит минимум oom_score_adj для собственного процесса.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: Рут

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 15]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+AdGuard будет отображать информацию об удаленных элементах HTML в журнале фильтрации.
+    
+  **pref.samsungpay.autopause.enable**
+
+AdGuard приостановит защиту, пока приложение Samsung Pay находится на переднем плане.
+
+  **pref.dns.bootstrap**
+   
+Начальная загрузка DNS для DNSProxy DoH и DoT серверов.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: `Начальная загрузка DNS` -> переформулируй пжлст

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 16]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+   
+ **pref.dns.fallback**
+   
+Резервный DNS-адрес.
+
+ **pref.dns.blocking.type**
+   
+Тип блока для фильтрации DNS.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: `Тип блока` -> Переформулируй

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 17]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+ **pref.notify.on.unknown.ca**
+
+Показывать уведомления о неизвестных CA.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: `CA`?

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 18]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+Enforce notification about paused protection even when notification icon is set to Disabled.
+
+  **pref.root.clear.youtube**
+  
+AdGuard will clear Youtube app data every on boot (ROOT is required).
+
+  **pref.root.set.oom_adj**
+   
+AdGuard will set minimum oom_score_adj for own process (Root is required).
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: Please, make `ROOT` -> `Root` or `Root` -> `ROOT`

File: 05.android/06.solving-problems/06.low-level-settings/docs.en.md [hunk 19]
```diff
@@ -38,3 +37,126 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
...
+
+List of the redirect ports.
+
+  **pref.boot.startup.delay**
+
+The initial protection startup delay (in seconds) after device boot up.
+
+  **pref.enforce.paused.notification**
+
+Enforce notification about paused protection even when notification icon is set to Disabled.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.en.md]: For Android version < Oreo

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 20]
```diff
@@ -30,4 +30,126 @@ visible: true
...
+
+Принудительное уведомление о приостановленной защите, даже если для значка уведомления установлено значение Выключено.
+
+  **pref.root.clear.youtube**
+  
+AdGuard будет очищать данные приложения Youtube каждый раз при загрузке (необходимы права ROOT).
+
+  **pref.root.set.oom_adj**
+   
+AdGuard установит минимум oom_score_adj для собственного процесса (необходимы права ROOT).
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: `минимум` -> `минимальный`

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 21]
```diff
@@ -30,4 +30,126 @@ visible: true
...
+  **pref.removed.html.log**
+
+AdGuard будет отображать информацию об удаленных элементах HTML в журнале фильтрации.
+    
+  **pref.samsungpay.autopause.enable**
+
+AdGuard приостановит защиту, пока приложение Samsung Pay находится на переднем плане (только в режиме VPN).
+
+  **pref.dns.bootstrap**
+  DNS-сервер начальной загрузки для DoH и DoT серверов.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: Лишний пробел

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 22]
```diff
@@ -30,4 +30,126 @@ visible: true
...
+
+Исключенные UID'ы.
+
+  **pref.vpn.capture**
+
+Включает отображение запросов VPN в журнале фильтрации, файл PCAP будет записан стеком TCP/IP.
+  
+  **pref.har.capture**
+
+Включает захват файла HAR.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: Переформулируй пжлст

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 23]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+Адрес интерфейса IPv6 TUN.
+
+   **pref.vpn.ipv6.force**
+   
+Если включено - AdGuard добавит IPv6-адрес в VPN, даже если нет активной сети IPv6.
+
+   **pref.vpn.ipv4.bypass**
+   
+Если включено - AdGuard специальный маршрут IPv4, чтобы просто обойти его.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: AdGuard `что` специальный маршрут?

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 24]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+Список маршрутов IPv4, исключенных из VPN и авто-прокси.
+   
+   **pref.ipv6.routes.excluded**
+   
+Список маршрутов IPv6, исключенных из VPN и авто-прокси.
+
+   **pref.excluded.uids**
+
+Исключенные UID-пакеты.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: UID `пакетов`

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 25]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+  **pref.vpn.capture**
+
+Включает отображение запросов VPN в журнале фильтрации, файл PCAP будет записан стеком TCP/IP.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: При чем тут журнал фильтрации?

File: 05.android/06.solving-problems/06.low-level-settings/docs.ru.md [hunk 26]
```diff
@@ -30,4 +30,127 @@ visible: true
...
+
+AdGuard приостановит защиту, пока приложение Samsung Pay находится на переднем плане (только в режиме VPN).
+
+  **pref.dns.bootstrap**
+
+DNS-сервер начальной загрузки для DoH и DoT серверов.
+   
+ **pref.dns.fallback**
+   
+Резервный DNS-адрес.
```
Comment (admitrevskiy) [inline 05.android/06.solving-problems/06.low-level-settings/docs.ru.md]: DNS сервер

**Pull request merged**